packages/client/src/comp/Contact.tsx

```diff
@@ -0,0 +1,232 @@
+      margin: calc(var(--vh) * 6) 0;
+      height: calc(var(--vh) * 20);
+      font-size: 1.1em;
+    }
+  }
+`
+
+export const Contact: React.FC = () => {
+  const [validEmail, setValidEmail] = React.useState(true)
+  const [validPhone, setValidPhone] = React.useState(true)
+  const [emailValue, setEmailValue] = React.useState('')
+  const [phoneValue, setPhoneValue] = React.useState('')
+  const [nameValue, setNameValue] = React.useState('')
+  const [msgValue, setMsgValue] = React.useState('')
+
+  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    // TODO send email
+    e.preventDefault()
+  }
+
+  /** Returns an element indicating that this field is not correct */
+  const isValid = (input: 'email' | 'phone') => {
+    if (input === 'email') {
+      if (!validEmail && emailValue) return <span>(Invalid)</span>
+    } else {
+      if (!validPhone && phoneValue) return <span>(Invalid)</span>
+    }
+    return null
+  }
+
+  /** Updates if an input (email/phone) is valid or not */
+  const onInputBlur = (input: 'email' | 'phone') => {
+    if (input === 'email') {
+      setValidEmail(emailRegExp.test(emailValue))
+    } else {
+      setValidPhone(phoneRegExp.test(phoneValue))
+    }
+  }
+
+  /** Updates input (email/name/phone) fields */
+  const onInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.currentTarget
+    switch (name) {
+      case 'name':
+        setNameValue(value)
+        break
+      case 'email':
+        setEmailValue(value)
+        setValidEmail(true)
+        break
+      case 'phone':
+        setPhoneValue(value)
+        setValidPhone(true)
+        break
+    }
+  }
+
+  const onMessageChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setMsgValue(e.currentTarget.value)
+  }
+
+  return <Wrapper>
+    <Headline>
+      <h1>Contact us</h1>
+      <p>
+        Questions about our process, our team or anything else? Get in touch!
+      </p>
+    </Headline>
+
+    <Form action="contact_us" onSubmit={onSubmit}>
+      <label htmlFor="name">Name</label>
+      <input
+        type="text"
+        name="name"
+        onChange={onInputChange}
+        value={nameValue}
+      />
+
+      <label htmlFor="email">Email {isValid('email')}</label>
+      <input
+        type="text"
+        name="email"
+        onChange={onInputChange}
+        onBlur={() => onInputBlur('email')}
+        value={emailValue}
+      />
+
+      <label htmlFor="phone">Phone Number {isValid('phone')}</label>
+      <input
+        type="text"
+        name="phone"
+        onChange={onInputChange}
+        onBlur={() => onInputBlur('phone')}
+        value={phoneValue}
+      />
+
+      <label htmlFor="message">Your Message</label>
+      <textarea
+        name="message"
+        cols={30}
+        rows={10}
+        onChange={onMessageChange}
+        value={msgValue}
+      />
+      <RoundedButton variant="flat" color='primary'>Submit</RoundedButton>
```

Comment:
Can we make this blue like the other buttons?  I couldn't figure out how to do it.